### ISSUE TITLE
Add more context to the GitHub events in the activity feed

### ISSUE THREAD

**Issue by JoshSmith** [presidential-innovation-fellows/code-gov-web#122]
The GitHub events in the "Activity" section of a repo on code.gov appear to be mostly empty and without context:

![screen shot 2016-11-03 at 6 54 51 pm](https://cloud.githubusercontent.com/assets/321667/19991929/0ab73b54-a1f7-11e6-85f4-21686a6283f0.png)

Pulling in some additional context here would be useful, although I might suggest eschewing the activity feed in favor of showing open issues or recently merged pull requests.

Not sure what your plans are with regards to this feature, so just take these suggestions as you will.

**Comment by lukad03** [comment]
@JoshSmith good point. We only touched the service of the Activity/Events data for Repos. Our hope is to make this more useful and to pull in more contextualized data, perhaps even organizing these events by type (i.e. Pull Request, Issues, etc.).

What kind of data would be most useful to pull in to get a high-level sense of this repo? We're doing our best not to get into the business of cloning the functionality of GitHub and are keeping the emphasis on the discovery experience, so any suggestions you have here are more than welcome.

**Comment by JoshSmith** [comment]
@lukad03 I figured you don't want to recreate GitHub's UI, which is why I would actually lean towards not pulling in much data here.

I guess the bigger question I have is what job you want this to perform. My guess is that you're looking to:

- Provide transparency into your code and your processes
- Give a good jumping off point for people who want to help

In this sense, the UX of an activity feed is usually to provide a dashboard for people in order to improve utility for regular use along with user retention inside the product. But if most of the action happens out in the project on GitHub, then I'm not sure I see how it solves the jobs you're higher code.gov to generally perform.

**Comment by jbjonesjr** [comment]
@JoshSmith important part here is that not all repos may be hosted on a GitHub/Lab/Bucket type platform that provides easy web visualization of activity data (👋 svn server running on someone's desktop). For that reason, I think providing some sort of details about recent activity here does make sense.

@lukad03 did this also derive from the conversation on how to tell if a project is `active` or has been `abandoned`? I thought exposing recent activity was one of the ways to expose that.

One comment I did want to make is translating some of these verbs into an easier to understand language `Push`, `IssueComment`,`Fork` might be a bit confusing for first time users (or those not familiar with Git or GitHub-like tools).

**Comment by DanielJDufour** [comment]
Hi, everyone.  Sounds like a great idea, but honestly I don't have the bandwidth at the moment to make it happen.  I'm putting this issue on hold until I complete my current slate of tasks (which'll take a couple months).  I'm also open to PR's and suggestions of perhaps pre-existing Angular components that can make developing this quicker.
